### PR TITLE
Fix order of pause and port in command line documentation.

### PR DESCRIPTION
**Pull request by andi5**

### CODE REVIEW

**Comment by jfrazelle** [comment]
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:

`git commit --amend -s --no-edit && git push -f`

**Comment by SvenDowideit** [comment]
nice catch! :) LGTM - @fredlf @jamtur01

File: docs/sources/reference/commandline/cli.md [hunk 1]
```diff
@@ -1310,6 +1310,22 @@ timestamp, for example `2014-09-16T06:17:46.000000000Z`, to each
 log entry. To ensure that the timestamps for are aligned the
 nano-second part of the timestamp will be padded with zero when necessary.
 
+## pause
+
+    Usage: docker pause CONTAINER
+
+    Pause all processes within a container
+
+The `docker pause` command uses the cgroups freezer to suspend all processes in
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: uses the cgroups freezer function

File: docs/sources/reference/commandline/cli.md [hunk 2]
```diff
@@ -1310,6 +1310,22 @@ timestamp, for example `2014-09-16T06:17:46.000000000Z`, to each
...
+a container.  Traditionally when suspending a process the `SIGSTOP` signal is
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: Traditionally,

File: docs/sources/reference/commandline/cli.md (hunk 1 above)
Comment (andi5) [inline docs/sources/reference/commandline/cli.md]: Looks like the documentation talks about the "cgroup freezer". I am not sure whether function is a correct term, though.

Comment (crosbymichael) [inline docs/sources/reference/commandline/cli.md]: As this is moving existing content I think we can look into changing wording later in another PR if you think function or another term is better

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**